packages/@sanity/form-builder/src/inputs/DateInputs/base/LazyTextInput.tsx

```diff
@@ -66,6 +66,7 @@ export const LazyTextInput = React.forwardRef(function LazyTextInput(
   return (
     <TextInput
       {...rest}
+      data-testid="date-input"
```

Comment:
Did we actually add any tests as part of this change? I think this would be better to pass at usage site instead of hard coding here. Depending on context you might want to give it different test-ids, but this change makes that not possible.

Reply:
Me and Rico tried, but wasn't able to figure out how to test this since the behaviour was actually correct in tests.